design/alameda/design.md

```diff
@@ -0,0 +1,34 @@
+## What is Alameda
+
+Alameda is an intelligent resource orchestrator for Kubernetes, providing the features of autonomous balancing, scaling, and scheduling by using machine learning. Alameda learns the continuing changes of computing resources from K8S clusters, predicts the future computing resources demands for pods and nodes, and intelligently orchestrates the underlying computing resources without manual configuration.
+
+For more details, please refer to https://github.com/containers-ai/Alameda
+
+## How Alameda works with Rook
+
+![integration](./architecture_rook_integration.png)
+(Note: CRD AlamedaResource is called AlamedaScaler and CRD AlamedaResourcePrediction is called AlamedaRecommendation from release 0.2)
+
+This figure illustrates Alameda's architecture and how it can work with Rook. The primary purpose of Alameda is to **recommend optimal computing resource configuration for Kubernetes**. To achieve that, users of Alameda (which is Rook here) specify which Pods require metrics prediction and resource configuration recommendation by creating *AlamedaResource* CRs ([example](https://github.com/containers-ai/alameda/blob/master/example/samples/nginx/alameda_deployment.yaml)). After some machine learning calculations, users can see metrics predictions and recommendations in the *AlamedaResourcePrediction* CRs. An example can be found [here](https://github.com/containers-ai/alameda/blob/master/docs/quickstart.md#example). Please note that starting from Alameda release 0.2, the metrics predictions will be store in a time-series DB instead of CRs due to performance and size considerations of etcd.
```

Comment:
the example AlamedaResource CRD link is broken (404): https://github.com/containers-ai/alameda/blob/master/example/samples/nginx/alameda_deployment.yaml

i'd recommend breaking up this paragraph to a new line per sentence, comments can be added by line, which is an entire paragraph right now, so it's difficult to make scoped comments.